change/@ni-nimble-components-a398657e-0eca-4a73-9948-71fd7466f4bd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The test coverage looks quite solid, but it'd probably be worth scanning [table-sorting.spec.ts](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/table/tests/table-sorting.spec.ts) to see if there are any corner cases worth adding coverage for. I don't think we need to duplicate every test there with a custom sort order variation, but here are some ideas that could be valuable:
1. a test that asserts the actual order that rows end up in. I believe existing tests only cover TanStack configuration, unless I missed something.
2. same as above, but with two rows with equal custom sort index (we decided we're fine with whatever TanStack does in this case, but it could be good to assert about that behavior so we're aware if it changes.
3. a custom sort order equivalent of `can sort with have field names containing "."`. I don't recall why we added that test but presumably a similar corner case exists for the new custom sort order field.
4. two columns that point to the same custom sort order field. Nothing in particular makes me think this would fail, just a case that could be goofy depending on implementation choices.
5. any other cases you can think of after scanning the existing tests (hierarchy, sort order, etc)

Reply:
I hadn't added tests for these cases since none of the table code changed at all, and I also didn't change any of the ways we interact with TanStack. We're exercising the table's code path for `columnInternals.operandDataRecordFieldName` changing, which we already have tests for. I can't think of anything special to test for, other than `columnInternals.operandDataRecordFieldName` being set to the correct value, and the mixin tests verify that.

Reply:
Thanks for that explanation. I think the argument that there are no table changes so we don't need table test changes is compelling, so I'm ok to close this with no action.